.circleci/config.yml

```diff
@@ -80,15 +107,33 @@ workflows:
   version: 2
   default:
     jobs:
-      - install
+      - install:
+          filters:
```

Comment:
Maybe we can have an anchor for that? :laughing: 
Just an idea...

Reply:
I hate yaml and would have had to look that up.